CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
+## 0.0.65
+
+- [DSLocationMessageBubble] Removing key from DS
```

Comment:
I think that this comment isn't necessary, because it's a completely new widget from 0.0.64, as stated below.